app/src/main/java/com/android/wardrobeManager/ClosetActivity.java

```diff
@@ -1,31 +1,78 @@
 package com.android.wardrobeManager;
 
 import android.content.Intent;
+import android.opengl.Visibility;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
+import android.widget.TextView;
 
 public class ClosetActivity extends AppCompatActivity {
 
+
+    boolean shirtsScrollVisible = false;
+    boolean shoesScrollVisible = false;
+    boolean shortsScrollVisible = false;
```

Comment:
These clothing items don't have to be plural. It looks kinda strange with 'Scroll' as the second word. Just make it 
`
boolean shirtScrollVisible = false;
boolean shoeScrollVisible = false;
boolean shortScrollVisible = false;`